chain/chain/src/types.rs

```diff
@@ -157,16 +156,20 @@ impl BlockEconomicsConfig {
     /// version higher than those changes are not overwritten and will instead
     /// respect the value defined in genesis.
     pub fn min_gas_price(&self, protocol_version: ProtocolVersion) -> Balance {
-        if self.genesis_protocol_version < MIN_PROTOCOL_VERSION_NEP_92 {
-            if protocol_version >= MIN_PROTOCOL_VERSION_NEP_92_FIX {
+        if self.genesis_protocol_version
+            < ProtocolFeature::MinProtocolVersionNep92.protocol_version()
```

Comment:
change pattern to `!ProtocolFeature::MinProtocolVersionNep92.enabled(self.genesis_protocol_version)`

Similarly below for `ProtocolFeature::MinProtocolVersionNep92Fix`